_posts/1-2-17-Video.md

```diff
@@ -7,9 +7,8 @@ section: UI toolkit
 
 # Video
 
-Video component is a component that can be used to render all types of Video items.  
-- If the source is Youtube or Vimeo, then the `NativeVideo` component plays the video  
-- In all other cases, the video is displayed in a WebView using `WebViewVideo` component  
+`Video` component can be used to render all types of Video items.  
+It renders a Video based on the source type. If source is an url to a web player the video is displayed in a `WebView`, if not, then a `Video` `HTML` element is displayed in the `WebView`.  
```

Comment:
I think this is a better form: If a source is `URL` to a web player, video is displayed in a `WebView`. If not, `video` `HTML` element is displayed in the `WebView`.